conf/evolutions/default/242.sql

```diff
@@ -0,0 +1,35 @@
+# --- !Ups
+DELETE FROM label_validation
+WHERE label_validation_id IN (
+  SELECT label_validation_id
```

Comment:
We've been using 4-space indents in previous evolutions files, so let's stick with that